browser/src/Input/KeyBindings.ts

```diff
@@ -119,6 +133,17 @@ export const applyDefaultKeyBindings = (oni: Oni.Plugin.Api, config: Configurati
 
     input.bind("<s-c-b>", "sidebar.toggle", isNormalMode)
 
+    // Explorer
+    input.bind("d", "explorer.delete.persist", isExplorerActive)
```

Comment:
What is the difference between `delete` and `delete.persist`?

Reply:
I think `delete.persist` keeps the deleted file around for some amount of time in a temp folder, such that you can undo it (as long as its under the file size limit).

`delete` deletes it straight away.

Basically...an Oni recycle bin for those who want it. For those who don't they can rebind `d` to `delete` and instantly delete. May be worth adding that to your minimal config? Though I'm not sure what NerdTree etc do tbh, its been a while.

Reply:
This seems fine to me, thanks @CrossR. Usually alternative actions I prefer the upcased char instead of Control, but that's fine.